contracts/src/chainlink/cairo/ocr2/aggregator.cairo

```diff
@@ -210,7 +203,7 @@ end
 # --- Configuration
 
 @event
-func config_set(
+func ConfigSet(
```

Comment:
Be careful to search the whole project. There are more instances where events are used in the relayer for example:

- https://github.com/smartcontractkit/chainlink-starknet/search?q=config_set
- https://github.com/smartcontractkit/chainlink-starknet/search?q=new_transmission